pkg/virtual/syncer/types.go

```diff
@@ -20,6 +20,12 @@ import (
 	apiresourcev1alpha1 "github.com/kcp-dev/kcp/pkg/apis/apiresource/v1alpha1"
 )
 
+const SyncerFinalizer string = "workloads.kcp.dev/syncer"
+
+func LocationClusterLabelName(locationName string) string {
```

Comment:
There is a constant for this in the API package now. Don't over-abstract with helpers. `SomeConstantPrefix + locationName` is totally fine.